.github/labeler.yml

```diff
@@ -1,57 +1,78 @@
-#
-# Add project labels to PRs
-# Invocation is done by .github/workflows/prlabeler.yml
-
-# changes in the documentation
-'Documentation':
-- changed-files:
-  - any-glob-to-any-file: 'book_source/**'
-  - any-glob-to-any-file: 'documentation/**'
-  - any-glob-to-any-file: 'CONTRIBUTING.md'
-  - any-glob-to-any-file: 'DEBUGING.md'
-  - any-glob-to-any-file: 'DEV-INTRO.md'
-  - any-glob-to-any-file: 'README.md'
-
-# Add 'Dockerfile' label to any changes in the docker directory
-'Dockerfile':
-- changed-files:
-  - any-glob-to-any-file: 'docker/**'
-
-
-# Add 'Website' label to any changes in the web directory
-'Website':
-- changed-files:
-  - any-glob-to-any-file: 'web/**'
-
-# Add 'Base' label to any changes in the base directory
-'Base':
-- changed-files:
-  - any-glob-to-any-file: 'base/**'
+# Auto-apply labels to PRs based on changed files
+# Triggered by .github/workflows/prlabeler.yml
+
+# Documentation
+'Documentation/User':
+  - changed-files:
+      - any-glob-to-any-file: 'book_source/**'
+      - any-glob-to-any-file: 'documentation/user_guides/**'
+
+'Documentation/Developer':
+  - changed-files:
+      - any-glob-to-any-file: 'CONTRIBUTING.md'
+      - any-glob-to-any-file: 'DEBUGGING.md'
+      - any-glob-to-any-file: 'DEV-INTRO.md'
+
+'Documentation/General':
+  - changed-files:
+      - any-glob-to-any-file: 'README.md'
+      - any-glob-to-any-file: 'documentation/**'
+
+# Models
+'Models/Climate':
+  - changed-files:
+      - any-glob-to-any-file: 'models/climate/**'
 
-# Add 'Models' label to any changes in the models directory
-'Models':
-- changed-files:
-  - any-glob-to-any-file: 'models/**'
+'Models/Biophysical':
+  - changed-files:
+      - any-glob-to-any-file: 'models/biophysical/**'
+
+'Models/Economics':
+  - changed-files:
+      - any-glob-to-any-file: 'models/economics/**'
 
-# Add 'Modules' label to any changes in the modules directory
+# Modules
 'Modules':
-- changed-files:
-  - any-glob-to-any-file: 'modules/**'
+  - changed-files:
+      - any-glob-to-any-file: 'modules/**'
 
-# Add 'GitHub Actions' label to any changes in the .github/workflows directory
-'GitHub Actions':
-- changed-files:
-  - any-glob-to-any-file: '.github/workflows/**'
+# Scripts
+'Scripts/R':
+  - changed-files:
+      - any-glob-to-any-file: '**/*.R'
```

Comment:
As I said in the comments to #3273, I don't think language-specific labels are very useful for this project because almost every PR will touch one or more R files while other languages tend to only show up in defined places, so the presence or absence of an "R" or "Python" label doesn't add much information for a human reader trying to decide whether a given PR needs their attention.